addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -255,7 +255,11 @@ export class WebsitePreview extends Component {
      */
     _isTopWindowURL({ host, pathname }) {
         const backendRoutes = ['/web', '/web/session/logout'];
-        return host !== window.location.host || (pathname && (backendRoutes.includes(pathname) || pathname.startsWith('/@/')));
+        return host !== window.location.host
+            || (pathname
+                && (backendRoutes.includes(pathname)
+                    || pathname.startsWith('/@/')
+                    || pathname.startsWith('/web/content')));
```

Comment:
This should be on its own commit (actually probably even on its own PR), it has nothing to do with the rest

Reply:
Moved this fix here: https://github.com/odoo/odoo/pull/102225